packages/models-library/src/models_library/payments.py

```diff
@@ -42,6 +43,8 @@ class InvoiceDataGet(BaseModel):
     stripe_price_id: StripePriceID
     stripe_tax_rate_id: StripeTaxRateID
     user_invoice_address: UserInvoiceAddress
+    user_name: str
```

Comment:
can yo please use a different name for this variable, something like `user_full_name` or `user_display_name` to distinguish from `user.name` that is an identifier